packages/jest-matcher-utils/src/index.js

```diff
@@ -151,18 +151,22 @@ export const matcherHint = (
   received: string = 'received',
   expected: string = 'expected',
   options: ?{
-    secondArgument?: ?string,
+    comment?: string,
     isDirectExpectCall?: boolean,
+    secondArgument?: ?string,
   },
```

Comment:
Can we set this to have a default value of `= {}` to avoid further `options &&` checks?